draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -418,15 +419,21 @@ In this case, the Issuer would use the following data in the Disclosures for the
 
 {{examples/address_only_structured/disclosures.md}}
 
-##### Option 3: Structured SD-JWT, only some properties selectively disclosable
-
 The Issuer may also make one sub-claim of `address` non-selectively disclosable and hide only the other sub-claims:
 
 <{{examples/address_only_structured_one_open/sd_jwt_payload.json}}
 
-In this case, the Issuer would issue the following Disclosures:
+There would be no Disclosure for `country` in this case.
+
+#### Option 3: Recursive SD-JWT
```

Comment:
i am slightly worried this title might be confusing..
would be more in favor of something like `SD-JWT with recursive Disclosures`?

Reply:
^ makes sense

Reply:
```suggestion
#### Option 3: SD-JWT with Recursive Disclosures
```